README.md

```diff
@@ -81,15 +78,23 @@ sudo docker ps -a
 ```
 If any of the LCP containers have *Exited*, then execute a command to list the logs for the failed container:
 ```bash
-sudo docker logs base-local_lcpserver_1
+docker-compose logs lcpserver
 ```
 This should present an error or condition indicating why the container exited. If all of the containers are running, you should be good to proceed.
 
 ### Upload a Book via the Front-end ###
-Enter the front-end's web interface. Given the default values we coded in the docker-compose.yaml file, enter:
+For local testing you'll need to adjust your [hosts file](/etc/hosts) and add the following string there:
+```
+127.0.0.1 testfrontend
+```
+It's required because `testfrontend` host name is used in the front-end source code.
+
+After that access the front-end in a web browser using the following address:
 ```bash
-http://192.168.1.253:8991
+http://localhost:8991
```

Comment:
One of the benefits of having the URLs specified by IP address here is that the `hosts` entry is not needed (either on the local machine, or one another computer on the same network). But I see your point. If/when we move to actual domains in the next iterations somewhere, either a local hosts entry or a DNS entry will be required.
   I was also initially thinking of the need outside a single host Docker stack of the lsdserver needing to be available publicly. The Docker host naming works in a single-stack compose deployment. But we'll change if/when we separate the services to different hosts (or, say, the ECS service in AWS).

Reply:
My main intention was to simplify local testing because `localhost` always works and doesn't depend on local configuration. Testing the frontend it from outside would require a user to know a a private IP inside the network or a public IP (if one exists) which might be difficult and require additional instructions

Reply:
No problem. We'll leave it at localhost for this branch of the repo.